syntax/internal/stdlib/stdlib.go

```diff
@@ -309,33 +315,33 @@ func yamlDecode(in string) (interface{}, error) {
 	return res, nil
 }
 
-func base64Decode(in string) (interface{}, error) {
+func base64Decode(in string) ([]byte, error) {
```

Comment:
These were unnecessarily untyped making it harder to read and reason about the code. I've given them a stronger type constraint to prevent bugs.